examples/stm32/xively_demo_stm32f4xx_nucleo_eth/SW4STM32/STM32F429ZI_NUCLEO_144/.cproject

```diff
@@ -187,6 +187,45 @@
 									<listOptionValue builtIn="false" value="STM32F429xx"/>
 									<listOptionValue builtIn="false" value="USE_STM32F4XX_NUCLEO_144"/>
 									<listOptionValue builtIn="false" value="USE_STM32F4XX_NUCLEO"/>
+									<listOptionValue builtIn="false" value="NO_DH"/>
+									<listOptionValue builtIn="false" value="NO_DES"/>
+									<listOptionValue builtIn="false" value="NO_DES3"/>
+									<listOptionValue builtIn="false" value="NO_DSA"/>
+									<listOptionValue builtIn="false" value="NO_HC128"/>
+									<listOptionValue builtIn="false" value="NO_MD4"/>
+									<listOptionValue builtIn="false" value="NO_OLD_TLS"/>
+									<listOptionValue builtIn="false" value="NO_PSK"/>
+									<listOptionValue builtIn="false" value="NO_PWDBASED"/>
+									<listOptionValue builtIn="false" value="NO_RC4"/>
+									<listOptionValue builtIn="false" value="NO_RABBIT"/>
+									<listOptionValue builtIn="false" value="NO_SHA512"/>
+									<listOptionValue builtIn="false" value="SINGLE_THREADED"/>
+									<listOptionValue builtIn="false" value="CUSTOM_RAND_GENERATE=xively_ssl_rand_generate"/>
+									<listOptionValue builtIn="false" value="HAVE_SNI"/>
+									<listOptionValue builtIn="false" value="HAVE_OCSP"/>
+									<listOptionValue builtIn="false" value="HAVE_CERTIFICATE_STATUS_REQUEST"/>
+									<listOptionValue builtIn="false" value="SMALL_SESSION_CACHE"/>
+									<listOptionValue builtIn="false" value="NO_CLIENT_CACHE"/>
+									<listOptionValue builtIn="false" value="WOLFSSL_SMALL_STACK"/>
+									<listOptionValue builtIn="false" value="WOLFSSL_USER_IO"/>
+									<listOptionValue builtIn="false" value="SIZEOF_LONG_LONG=8"/>
+									<listOptionValue builtIn="false" value="NO_WRITEV"/>
+									<listOptionValue builtIn="false" value="NO_WOLFSSL_DIR"/>
+									<listOptionValue builtIn="false" value="USE_FAST_MATH"/>
+									<listOptionValue builtIn="false" value="TFM_TIMING_RESISTANT"/>
+									<listOptionValue builtIn="false" value="NO_DEV_RANDOM"/>
+									<listOptionValue builtIn="false" value="NO_FILESYSTEM"/>
+									<listOptionValue builtIn="false" value="USE_CERT_BUFFERS_2048"/>
+									<listOptionValue builtIn="false" value="NO_ERROR_STRINGS"/>
+									<listOptionValue builtIn="false" value="NO_SERVER"/>
+									<listOptionValue builtIn="false" value="USER_TIME"/>
+									<listOptionValue builtIn="false" value="HAVE_ECC"/>
+									<listOptionValue builtIn="false" value="HAVE_TLS_EXTENSIONS"/>
+									<listOptionValue builtIn="false" value="HAVE_AESGCM"/>
+									<listOptionValue builtIn="false" value="ALT_ECC_SIZE"/>
+									<listOptionValue builtIn="false" value="NO_FILESYSTEM"/>
+									<listOptionValue builtIn="false" value="WC_RSA_BLINDING"/>
+									<listOptionValue builtIn="false" value="WOLFSSL_NO_SOCK"/>
 								</option>

```

Comment:
I'm not sure, but as my previous note stated, the ST Project, Build and Tutorial process are broken.  I've created a story to sort it all out as it's not part of this PR:  https://jira.3amlabs.net/browse/XCL-3037